butlarr/config/services.py

```diff
@@ -1,30 +1,54 @@
 from . import CONFIG
 from ..services.radarr import Radarr
 from ..services.sonarr import Sonarr
+from ..services.bazarr import Bazarr
 
 APIS = CONFIG["apis"]
 SERVICES = []
 
 for service in CONFIG["services"]:
+    name = service.get("name")
     service_type = service["type"]
     commands = service["commands"]
     api_config = APIS[service["api"]]
-
-    if service_type == "Radarr":
+    addons = service.get("addons", [])
+        
+    if service_type == "Radarr":               
         SERVICES.append(
             Radarr(
                 commands=commands,
                 api_host=api_config["api_host"],
-                api_key=api_config["api_key"]
+                api_key=api_config["api_key"],
+                name=name,
+                addons=addons
             )
         )
+
     elif service_type == "Sonarr":
         SERVICES.append(
             Sonarr(
                 commands=commands,
                 api_host=api_config["api_host"],
-                api_key=api_config["api_key"]
+                api_key=api_config["api_key"],
+                name=name,
+                addons=addons
+            )
+        )
+
+    elif service_type == "Bazarr":
+        SERVICES.append(
+            Bazarr(
+                commands=commands,
+                api_host=api_config["api_host"],
+                api_key=api_config["api_key"],
+                name=name,
+                addons=addons
             )
         )
+
     else:
-        assert False, "Unsupported service type!"
+        assert False, f"Unsupported service type {service_type}!"
+
+for service in SERVICES:
```

Comment:
Why is this in a separate loop?
Can't the service constructor do this automatically?

Reply:
I moved the logic to the services.py file, may need to optmize it a bit, couldn't use your code suggestion as is but will take a look at it again later with more time to spare